frontend/src/components/Simulator/TransactionItem.vue

```diff
@@ -55,7 +56,7 @@ const handleSetTransactionAppeal = () => {
 watch(
   () => props.transaction.status,
   (newStatus) => {
-    if (newStatus !== 'ACCEPTED') {
+    if (newStatus !== 'ACCEPTED' && newStatus !== 'UNDETERMINED') {
```

Comment:
@kstroobants what's the reason of this check? can't we rely in the appeal property of the transaction?

Reply:
Solved in 28c842c57e7fb8d7f314ef655ddb8b47a4b89347